index.html

```diff
@@ -1,527 +1,729 @@
-                                </ul>
-                            </div>
-                        </div>
+                      <div class="lang-list-frame">
+                        <img
+                          src="assets/img/javascript.png"
+                          alt="JavaScript Photo"
+                        />
+                        <h3>JavaScript</h3>
+                      </div>
                     </li>
                     <li>
-                    <div class="main-frame">
-                        <div class="frame-header">
-                            <div class="header-txt">
-                                <p>Frameworks</p>
-                            </div>
-                            <button class="normal-btn">
-                                <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                    <path fill-rule="evenodd" clip-rule="evenodd" d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z" fill="#505F79"/>
-                                </svg>
-                            </button>
-                        </div>
-                    </div>
-
+                      <div class="lang-list-frame">
+                        <img src="assets/img/html.png" alt="JavaScript Photo" />
+                        <h3>HTML</h3>
+                      </div>
                     </li>
                     <li>
-                        <div class="main-frame">
-                            <div class="frame-header">
-                                <div class="header-txt">
-                                    <p>Skills</p>
-                                </div>
-                                <button class="normal-btn">
-                                    <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
-                                        <path fill-rule="evenodd" clip-rule="evenodd" d="M9.29289 5.43962C9.68342 5.0491 10.3166 5.0491 10.7071 5.43962L16.7071 11.4396C17.0976 11.8301 17.0976 12.4633 16.7071 12.8538L10.7071 18.8538C10.3166 19.2444 9.68342 19.2444 9.29289 18.8538C8.90237 18.4633 8.90237 17.8301 9.29289 17.4396L14.5858 12.1467L9.29289 6.85384C8.90237 6.46331 8.90237 5.83015 9.29289 5.43962Z" fill="#505F79"/>
-                                    </svg>
-                                </button>
-                            </div>
-                        </div>
+                      <div class="lang-list-frame">
+                        <img src="assets/img/css.png" alt="JavaScript Photo" />
+                        <h3>CSS</h3>
+                      </div>
                     </li>
```

Comment:
- [ ] The language icons should be aligned horizontally, kindly fix this so your design is identical to the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220708_053828](https://user-images.githubusercontent.com/25227022/177918329-fcf5a58b-6d22-4ed4-a9ed-cc02d83a490a.png)|![Screenshot_20220708_053809](https://user-images.githubusercontent.com/25227022/177918339-06dfc496-9b7e-4485-b7a6-cd419f5d9d21.png)|